### ISSUE TITLE
Input not announced in braille panel

### ISSUE THREAD

**Issue by msviolafangirl** [musescore/MuseScore#20554]
### Issue type

UX/Interaction bug

### Bug description

When in the braille panel, pressing n results in no screen reader feedback about whether the user is in note input mode or normal mode. This is not an issue after one gets to the normal entry window by pressing shift+tab.

### Steps to reproduce

With screen reader enabled:
1. Go to the braille panel.
2. Press n. Notice that nothing is spoken.
3. Shift+tab to normal note entry area. Press n. Notice that either normal mode or note input mode is spoken.

### MuseScore Version

4.2

### Regression

No.

### Operating system

Windows 10

**Comment by shubham-shinde-442** [comment]
I would like to work on this issue.

**Comment by shoogle** [comment]
@shubham-shinde-442, I'm not sure why `NotationActionController` uses the `"note-input"` action to determine whether we're currently in note input mode when it already knows this from a few lines [above](https://github.com/musescore/MuseScore/blob/c21b364b466f28ea2896d6ff2bde0ef7a1f6a98b/src/notation/internal/notationactioncontroller.cpp#L667-L671). I suppose the call to `noteInput->startNoteInput()` could fail, so checking the status of the action code explicitly might be more reliable that just assuming the call succeeded.

Anyway, in `NotationBraille` it looks like we're [already telling](https://github.com/musescore/MuseScore/blob/8b8ea79ff1b4e1427c74bf2586273e249d94c47e/src/braille/internal/notationbraille.cpp#L863-L876) the accessibility system that we're entering braille input mode, so the real question is why isn't the screen reader saying anything?

My guess is it's to do with the fact the braille TextArea has focus rather than an element in the score view. Notice in [`NotationAccessibility::setTriggeredCommand()`](https://github.com/musescore/MuseScore/blob/8b8ea79ff1b4e1427c74bf2586273e249d94c47e/src/notation/internal/notationaccessibility.cpp#L115-L123) it associates the command message with a dummy item in the score view. That's probably not going to work when focus is outside the score view.

You probably need to modify `setTriggeredCommand()` to associate the message with an object elsewhere in the accessibility tree. Possibly the TextArea itself should be temporarily renamed to include the command message "Braille input mode", or you could transfer focus to a dummy panel that is named like the command message.

**Comment by shubham-shinde-442** [comment]
Thanks @shoogle .

**Comment by bkunda** [comment]
@shubham-shinde-442 are you still working on this? If so, I'll re-classify it as "in progress" 🙂

**Comment by shubham-shinde-442** [comment]
@bkunda No, I'm currently busy in university exams. If someone wants to work on this issue, please go ahead. Otherwise, I will fix it within a month 🙂

**Comment by bkunda** [comment]
@shubham-shinde-442 I think this one might still be available for you if you have time (hope your university exams went well, btw!). 
@shoogle FYI.

**Comment by shubham-shinde-442** [comment]
@bkunda Sure! I'll create PR soon 🙂

**Comment by jmwmusic** [comment]
A couple of us have tested this and with NVDA in 4.6 there is still no speech feedback when you press N while on the Braille window, so no way of knowing what mode you are in without tabbing on and off it and pressing N again.  Can this please be re-opened, if not I will open a new issue and reference this one.
